base/sparse/linalg.jl

```diff
@@ -946,17 +946,17 @@ function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
     a, b = size(X)
     n, p = vardim == 1 ? (a, b) : (b, a)
 
-    # Cov(X) = E[(X-μ)'(X-μ)]
-    # = X'X - X'μ
+    # Cov(X) = E[(X-μ)(X-μ)'] = E(XX') - μμ'
 
     # Compute X'X using sparse matrix operations
     out = Matrix(Base.unscaled_covzm(X, vardim))
 
-    # Compute X'μ
+    # Compute (estimate of) nμ
     sums = sum(X, vardim)
+
+    # Subtract nμμ' from X'X
```

Comment:
I have enough stats background to have seen (and taught...) covariance of matrices before and the notation collision is still confusing, especially when trying to describe code